CMakeLists.txt

```diff
@@ -6,8 +6,9 @@ set(NGX_OTEL_NGINX_BUILD_DIR ""
 set(NGX_OTEL_NGINX_DIR "${NGX_OTEL_NGINX_BUILD_DIR}/.."
     CACHE PATH "Nginx source dir")
 
-set(NGX_OTEL_FETCH_DEPS ON CACHE BOOL "Download dependencies")
-set(NGX_OTEL_PROTO_DIR  "" CACHE PATH "OTel proto files root")
+set(NGX_OTEL_FETCH_DEPS ON  CACHE BOOL "Download dependencies")
+set(NGX_OTEL_PROTO_DIR  ""  CACHE PATH "OTel proto files root")
+set(NGX_OTEL_DEV        OFF CACHE BOOL "Dev mode with stricter compiler diagnostics")
```

Comment:
"Enable stricter compiler diagnostics" or may be just "Enforce compiler warnings"?